bin/create-release

```diff
@@ -24,8 +24,8 @@ def git_generate_changelog
   end
 
   relevant_matches = matches.select do |match|
-    # ignore merges from staging; those represent DTTs
-    match['branch'] != 'staging'
+    # ignore merges from our various server branches
+    !match.nil? && !['staging', 'test', 'levelbuilder'].include?(match['branch'])
```

Comment:
Oh, I can see that. Particularly because we link to the contents of the scoop itself, it makes it easy to look for particular changes.